src/openqaoa-qiskit/backends/qaoa_qiskit_qpu.py

```diff
@@ -95,19 +95,11 @@ def __init__(
             False,
             None,
         ]:
-            if type(self.device).__name__ == "DeviceAzure":
-                raise Exception(
-                    "Connection to Azure was made. Error connecting to the specified backend."
-                )
-            else:
-                raise Exception(
-                    "Connection to IBMQ was made. Error connecting to the specified backend."
-                )
+            raise Exception("Connection to {} was made. Error connecting to the specified backend.".format(self.device.device_location.upper()))
```

Comment:
@shahidee44 what should be kept here?

Reply:
The new exception is better and doesnt require hardcoding

Reply:
Ok! Marking this resolved then :)